### ISSUE TITLE
`config nat feature enable` is blocked management port incoming traffic

### ISSUE THREAD

**Issue by iMasaruOki** [Azure/sonic-buildimage#4746]
**Description**
`config nat feature enable` is blocked management port incoming traffic e.g. cannot login by ssh.
outgoing traffic (and response) are passed.

**Steps to reproduce the issue:**
1. `sudo config nat feature enable`
2. ping or ssh from other host to management port.

**Describe the results you received:**

no response or connection timed out.

**Describe the results you expected:**

receive icmp reply and ssh login succeeded.

**Additional information you deem important (e.g. issue happens only occasionally):**

In `sonic-swss/cfgmgr/natmgr.cpp:atMgr::setFullConeDnatIptablesRule()`,
addi fullcone DNAT rule for all interfaces (includes management port) into kernel.

My quick hack,
- commented out adding fullcone NAT rule
- or, `-i eth0` is added to iptables command line.

**Output of `show version`:**

 ```
SONiC Software Version: SONiC.master.307-f31eabb5
Distribution: Debian 10.4
Kernel: 4.19.0-6-amd64
Build commit: f31eabb5
Build date: Wed Jun 10 13:20:12 UTC 2020
Built by: johnar@jenkins-worker-7

Platform: x86_64-delta_ag9032v2a-r0
HwSKU: Delta-ag9032v2a
 ```

**Comment by AkhileshSamineni** [comment]
@iMasaruOki This issue is seen because of supported fullcone NAT patch file is not yet present.  We are working on porting the supported fullcone NAT patch file for 4.19 kernel version, for time being please use 201911 branch for NAT feature.

**Comment by iMasaruOki** [comment]
It looks fixed by commit c79783003d1ec615eed50135bd56dbb793794128
Thank you.
